ovs/extensions/healthcheck/helpers/network.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2016 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+import socket
+
+
+class NetworkHelper(object):
+
+    @staticmethod
+    def is_port_listening(result_handler, process_name, port, ip):
+        """
+        Checks the port connection of a process
+        :param result_handler: logging object
+        :type result_handler: ovs.extensions.healthcheck.result.HCResults
+        :param process_name: name of a certain process running on this local machine
+        :type process_name: str
+        :param port: port where the service is running on
+        :type port: int
+        :param ip: ip address to try
+        :type ip: str
+        """
+        result_handler.info('Checking port {0} of service {1} ...'.format(port, process_name), '_is_port_listening')
+        if NetworkHelper.check_port_connection(port, ip):
```

Comment:
Function only called upon once, merge in here or if kept as separate function, make private function